Sources/libcggen/BCCGGenerator.swift

```diff
@@ -0,0 +1,330 @@
+      return byteCommand(6, rect, rx, ry)
+    case let .addArc(center, radius, startAngle, endAngle, clockwise):
+      return byteCommand(7, center, radius, startAngle, endAngle, clockwise)
+    case .closePath:
+      return byteCommand(8)
+    case .replacePathWithStrokePath:
+      return byteCommand(9)
+    case let .lines(lines):
+      return byteCommand(10, lines)
+    case let .clip(rule):
+      return byteCommand(11, rule)
+    case let .clipToRect(rect):
+      return byteCommand(12, rect)
+    case let .dash(pattern):
+      return byteCommand(13, pattern)
+    case let .fill(rule):
+      return byteCommand(14, rule)
+    case let .fillEllipse(rect):
+      return byteCommand(15, rect)
+    case .stroke:
+      return byteCommand(16)
+    case let .drawPath(mode):
+      return byteCommand(17, mode)
+    case let .addEllipse(rect):
+      return byteCommand(18, rect)
+    case let .concatCTM(transform):
+      return byteCommand(19, transform)
+    case let .flatness(f):
+      return byteCommand(20, f)
+    case let .lineWidth(width):
+      return byteCommand(21, width)
+    case let .lineJoinStyle(lineJoin):
+      return byteCommand(22, lineJoin)
+    case let .lineCapStyle(cap):
+      return byteCommand(23, cap)
+    case let .colorRenderingIntent(intent):
+      return byteCommand(24, intent)
+    case let .globalAlpha(alpha):
+      return byteCommand(25, alpha)
+    case let .strokeColor(color):
+      return byteCommand(26, color)
+    case let .fillColor(color):
+      return byteCommand(27, color)
+    case let .linearGradient(name, options):
+      return byteCommand(
+        28,
+        gradientsIds[name]!,
+        options.startPoint,
+        options.endPoint,
+        options.endPoint
+      )
+    case let .radialGradient(name, options):
+      return byteCommand(
+        29,
+        gradientsIds[name]!,
+        options.startCenter,
+        options.startRadius,
+        options.endCenter,
+        options.endRadius,
+        options.options
+      )
+    case let .linearGradientInlined(gradient, options):
+      return byteCommand(
+        30,
+        gradient,
+        options.startPoint,
+        options.endPoint,
+        options.options
+      )
+    case let .radialGradientInlined(gradient, options):
+      return byteCommand(
+        31,
+        gradient,
+        options.startCenter,
+        options.startRadius,
+        options.endCenter,
+        options.endRadius,
+        options.options
+      )
+    case let .subrouteWithName(name):
+      return byteCommand(32, subroutesIds[name]!)
+    case let .shadow(shadow):
+      return byteCommand(33, shadow)
+    case let .blendMode(mode):
+      return byteCommand(34, mode)
+    case .beginTransparencyLayer:
+      return byteCommand(35)
+    case .endTransparencyLayer:
+      return byteCommand(36)
+    case let .composite(steps):
+      return generateSteps(steps: steps)
+    case .endPath, .fillColorSpace, .strokeColorSpace:
```

Comment:
`.endPath` is rather important, please leave a comment that it will be fixed in future. And possibly leave a `fatalError()` for `fillColorSpace` and `strokeColorSpace`?